pinot-core/src/main/java/com/linkedin/pinot/core/query/selection/SelectionOperatorUtils.java

```diff
@@ -420,9 +421,25 @@ public static DataTable getDataTableFromRows(@Nonnull Collection<Serializable[]>
   @Nonnull
   public static SelectionResults renderSelectionResultsWithoutOrdering(@Nonnull List<Serializable[]> rows,
       @Nonnull DataSchema dataSchema, @Nonnull List<String> selectionColumns) {
+    // TODO: remove the code for backward compatible after server updated to the latest code.
```

Comment:
I suggest to create a jira ticket that has the version number of the server that must be deployed everywhere before we can remove this code.